CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# Release 0.4.2
+- [GH-1080] [Double-submit] Use updated environments.clj with gotc-prod being wgs-prod ([#251](https://github.com/broadinstitute/wfl/pull/251))
+- [GH-1082] fix wrong cromwell queried in `wgs/skip-workflow?` ([#252](https://github.com/broadinstitute/wfl/pull/252))
+- make error logging less noisy ([#249](https://github.com/broadinstitute/wfl/pull/249))
+- [GH-1078] use updated environments from pipeline_config ([#248](https://github.com/broadinstitute/wfl/pull/248))
+- open 0.4.2 for fixes
```

Comment:
nit: maybe we could take this out?

Reply:
no. I think honesty is the best policy here